### PR TITLE
doc: adding base for code of conduct doc

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

File: code-of-conduct [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (l-mb) [inline code-of-conduct:64]: Special group on Slack? Should we set up a mailing list?

Comment (asettle) [reply code-of-conduct:64]: I think this is where a mailing list would come in handy. Also, to add to that, we don't really have clear definitions of who consitutes "community leader" :eyes:

File: code-of-conduct [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time. No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a permanent ban.
+
+### 4. Permanent Ban
+
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior,  harassment of an
```
Comment (ml8mr) [inline code-of-conduct:110]:
```suggestion
standards, including sustained inappropriate behavior, harassment of an
```

**Comment by asettle** [comment]
Superceded by https://github.com/aquarist-labs/aquarium/pull/198 (to use appropriate forking)

**Pull request closed without merging**